tools/autograd/derivatives.yaml

```diff
@@ -1661,8 +1661,8 @@
   output: grad * grad_output * (-2 * output + 1)
 
 - name: tanh_backward(Tensor grad_output, Tensor output) -> Tensor
-  grad_output: tanh_backward(grad, output)
-  output: -2 * output * grad * grad_output
+  grad_output: tanh_backward(grad, output.conj())
+  output: grad.conj() * (-2 * output.conj() * grad_output)
```

Comment:
Is the re-ordering important here?

Reply:
no I just re-ordered it because I think it's nice to follow a general convention of `grad * ...` whenever possible for better code readability.

Reply:
It looks ok here. We just need to be careful as some of the orderings are important for numerical precision. But this one should be fine.